app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -71,17 +72,12 @@ class ConversationViewModel @Inject constructor(
         }
 
         viewModelScope.launch {
-            getConversationDetails(conversationId!!).let {
-                when (it) {
-                    is GetConversationDetailsUseCase.Result.Failure -> {
-                        TODO("unhandled error case")
-                    }
-                    is GetConversationDetailsUseCase.Result.Success -> {
-                        it.convFlow.collect { conversation ->
-                            conversationViewState = conversationViewState.copy(conversationName = conversation.name ?: "Some Name")
-                        }
-                    }
+            observeConversationDetails(conversationId!!).collect { conversationDetails ->
+                val conversationName = when (conversationDetails) {
+                    is ConversationDetails.OneOne -> conversationDetails.otherUser.name.orEmpty()
```

Comment:
if I'm getting it right, I think OneOnOne is much clear than OneOne!

Reply:
Yeaaah. I started preferring `OneOnOne` too after `OneOne` was already merged 😢 

I guess another simple renaming PR can be raised in Kalium later 😛

Reply:
All good buddy!